src/types/plugin-input.ts

```diff
@@ -15,7 +15,329 @@ export const pluginSettingsSchema = T.Object(
     devConfigPath: T.String({ default: ".github/.ubiquity-os.config.dev.yml" }),
     model: T.String({
       default: "deepseek/deepseek-r1-0528:free",
-      examples: ["deepseek/deepseek-r1-0528:free", "anthropic/claude-3.7-sonnet", "openai/gpt-4o"],
+      examples: [
```

Comment:
This is cool because I imagine when we use the @ubiquityos syntax we can fuzzy match the request i.e. "change to the latest deepseek reasoning model"